packages/manager/modules/carrier-sip/src/cdr/cdr.controller.js

```diff
@@ -0,0 +1,86 @@
+import capitalize from 'lodash/capitalize';
+import get from 'lodash/get';
+import range from 'lodash/range';
+
+import { GET_RECORD_NUMBER_OF_YEARS, POLLER_TIMEOUT } from './cdr.constants';
+
+export default class CarrierSipCdrCtrl {
+  /* @ngInject */
+  constructor(
+    $timeout,
+    $translate,
+    $window,
+    OvhApiTelephony,
+    TucToast,
+  ) {
+    this.$timeout = $timeout;
+    this.$translate = $translate;
+    this.$window = $window;
+    this.OvhApiTelephony = OvhApiTelephony;
+    this.TucToast = TucToast;
+  }
+
+  $onInit() {
+    const currentYear = moment().year();
+    this.months = moment.months().map(capitalize);
+    this.month = null;
+    this.years = range(
+      currentYear,
+      currentYear - GET_RECORD_NUMBER_OF_YEARS,
+      -1,
+    );
+    this.year = null;
+    this.poller = null;
+  }
+
+  /**
+   * Get the Call Detail Records of your Carrier SIP service
+   * {@link https://api.ovh.com/console/#/telephony/%7BbillingAccount%7D/carrierSip/%7BserviceName%7D/cdrs#GET}
+   * @return {Promise}
+   */
+  getCallDetailRecords() {
+    const { billingAccount, serviceName } = this;
+    const month = this.months.indexOf(this.month) + 1;
+
+    const tryGetTelephonyDocument = () => this.OvhApiTelephony
+      .CarrierSip()
+      .Cdrs()
+      .v6()
+      .get({
+        billingAccount,
+        serviceName,
+        month: `${this.year}-${month < 10 ? `0${month}` : month}`,
```

Comment:
This should be handled by the API 😅 

The standard way to do this is 
```suggestion
        month: `${this.year}-${month.toLocaleString(undefined, {minimumIntegerDigits: 2})}`,
```

Reply:
Actually, toLocaleString requires a mandatory valid `locale` as first parameter